readers/openapi.yml

```diff
@@ -140,6 +147,55 @@ components:
       schema:
         type: string
       required: false
+    Value:
+      name: value
+      description: SenML message value.
+      in: query
+      schema:
+        type: string
+      required: false
+    V:
+      name: v
+      description: SenML message value.
+      in: query
+      schema:
+        type: string
+      required: false
+    Vb:
+      name: vb
+      description: SenML message bool value.
+      in: query
+      schema:
+        type: boolean
+      required: false
+    Vs:
+      name: vs
+      description: SenML message string value.
+      in: query
+      schema:
+        type: string
+      required: false
+    Vd:
+      name: vd
+      description: SenML message data value.
+      in: query
+      schema:
+        type: string
+      required: false
+    From:
+      name: from
+      description: SenML message time in nanoseconds (int64).
```

Comment:
Is this Unix time?

Reply:
This is `time.Now().UnixNano()`. We should maybe use float64 here and convert it before the DB query.

Reply:
Then we should mention this in the description:
```
description: SenML message time in nanoseconds in Unix time format (int64).
```